### ISSUE TITLE
Uncaught TypeError: Cannot read property 'rss' of null

### ISSUE THREAD

**Issue by Jckarlos92** [enginkizil/FeedEk#41]
I'm trying to adapt my old code (which worked nicely when the Google's API was working) and it's giving me this error. 

I replaced my old FeedEk.js and I didn't change anything from the .FeedEk constructor

Could you have an idea why is this happening? These are the lines where the Google Console shows the error

``` javascript
    if (!(data.query.results.rss instanceof Array)) {
        data.query.results.rss = [data.query.results.rss];
    }
```

**Comment by enginkizil** [comment]
Hi, can you write your feed source url for test?

**Comment by Jckarlos92** [comment]
Hi!

It's an internal link (from the company I work for) so you won't get access to that.

Nonetheless, to get the rss news, it uses a php script that receives parameters throught the URL and I don't know if that's causing the error.

The link would be something like this

```javascript
   https://company.northernlight.com/rss.php?alertId=6931 
```

Hope that helps!

**Comment by BradGunnerSGT** [comment]
Having a similar issue, I can share my URL:  http://rssfeeds.utsa.edu/default.aspx?Feed=ASAP

FeedEk 3.0 and JQuery 2.1.4

**Comment by enginkizil** [comment]
Hi BradGunnerSGT;
FeedEk uses feednormalizer table in YQL. So when I try your feed url in YQL console with feednormalizer, i got an error "Could not parse feed data. Invalid rss_2.0 feed, missing channel link"

It seems your feed format is invalid for RSS specification.

**Comment by enginkizil** [comment]
@Jckarlos92 I advice you to test it on YQL console. Maybe your feed source format is invalid for RSS specification too.

**Comment by BradGunnerSGT** [comment]
It appears to work fine for me using feednormalizer as well as rss.

![screen shot 2015-12-11 at 6 42 05 pm](https://cloud.githubusercontent.com/assets/2065548/11758928/ec092568-a036-11e5-993f-eaa3f7b129ce.png)

**Comment by enginkizil** [comment]
@BradGunnerSGT FeedEk use this query 
SELECT * FROM feednormalizer WHERE output="rss_2.0" AND url ="http://rssfeeds.utsa.edu/default.aspx?Feed=ASAP" 

Result is: Could not parse feed data. Invalid rss_2.0 feed, missing channel link

Error occured because of FeedEk query try to make it rss 2.0 standart as output. Your feed source format is missing channel link.

**Comment by BradGunnerSGT** [comment]
Ok, that makes sense. I contacted the team that provides that RSS service and they have fixed the feed to fully conform to the RSS 2.0 specs. It's working great now!

**Comment by enginkizil** [comment]
YQL API is retired on 3th of January 2019. FeedEk v3.1.0 released and YQL API is not using anymore. Please use latest version of FeedEk v.3.1.0
@BradGunnerSGT @Jckarlos92
